Bitbucket.Authentication/Authentication.cs

```diff
@@ -87,25 +87,40 @@ public override void DeleteCredentials(TargetUri targetUri, string username)
         {
             BaseSecureStore.ValidateTargetUri(targetUri);
 
-            Trace.WriteLine("BitbucketAuthentication::DeleteCredentials");
+            Trace.WriteLine($"BitbucketAuthentication::DeleteCredentials for {targetUri.ActualUri}");
```

Comment:
Given that `Trace.WriteLine` already prints the method name, would it be more terse/meaningful if something else were printed, like `$"Deleting BitBucket credentials for {targetUri.ActualUri)?"